config/config.php

```diff
@@ -676,20 +676,36 @@
 		],
 	],
 	'beta' => [
+		'30' => [
+			'100' => [
+				'latest' => '30.0.3 RC1',
+				'internalVersion' => '30.0.3.0',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-30.0.3rc1.zip',
+				'web' => 'https://docs.nextcloud.com/server/30/admin_manual/maintenance/upgrade.html',
+				'eol' => false,
+				'minPHPVersion' => '8.1',
+				'signature' => 'VRjPeAIFlYCSZyWrdQVN5z7kU9uq/nMHE7ym691ru1Anu5QKlEBYDaSDEEt/Je/Q
+jea7xxnggPno7pAO84tK+F2ZN5dZ9o7ZH9XmleuW1E/QGU+G6guHlh4ZutAtw+eb
+JiHhzO+m3SucttavXKR4FiIxHhxOj/Ih5gp2sQd4iE+6JwgKgl1R2UiP9J43LGtJ
+ExTXwL2A0E8zfgBOHWDhNXB4YhjdFdzi+d+Dw+YV1eNswy+hXB3ZQd1t+96891T9
+RNlo41rfyZUMLPgdfS1Yrlw4JjDgwtdA4P1eSG/jNi3mEmFwwmkH5BWwgNtOjUPq
+ARAlASKdZcVbvofv7UKS3w==',
+			],
+		],
 		'29.0.9.1' => [
 			'100' => [
-				'latest' => '30.0.2',
-				'internalVersion' => '30.0.2.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-30.0.2.zip',
+				'latest' => '30.0.3 RC1',
```

Comment:
wouldn't latest stable 29 not point to latest RC 29?

Reply:
Or I am reviewing too early.... xD

Reply:
latest 29rc1 isn't built yet :grin: